build/e2e.gulp.js

```diff
@@ -29,11 +31,31 @@
   });
 
   gulp.task('coverage-combine', function (cb) {
-    var combine = require('istanbul-combine');
```

Comment:
Can we remove this as a dev dependency as well - I don't think its used anywhere else?

Reply:
The diff is a bit odd, the line's just moved further down

Reply:
Ah yes. See it now